crates/solvers-dto/src/auction.rs

```diff
@@ -33,11 +33,22 @@ pub struct Order {
     #[serde_as(as = "HexOrDecimalU256")]
     pub sell_amount: U256,
     #[serde_as(as = "HexOrDecimalU256")]
+    pub full_sell_amount: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
     pub buy_amount: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
+    pub full_buy_amount: U256,
+    pub fee_policies: Option<Vec<FeePolicy>>,
+    pub valid_to: u32,
     pub kind: Kind,
+    pub receiver: Option<H160>,
+    pub owner: H160,
     pub partially_fillable: bool,
+    pub pre_interactions: Vec<model::interaction::InteractionData>,
+    pub post_interactions: Vec<model::interaction::InteractionData>,
```

Comment:
This is still missing `app_data`, fee_amount` (😱), `sell_token_source`, `buy_token_destination`.

Reply:
aha, so everything is needed? got it

Reply:
btw what is `buy_token_destination`, don't you mean `buy_token_balance`?

Reply:
Yeah, in the old code it's called `buy_token_balance` but in the new code `buy_token_destination` IIRC. `buy_token_destination` makes more sense IMO.